packages/components/src/notice/index.js

```diff
@@ -7,17 +7,21 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
+import Dashicon from '../dashicon';
 import { __ } from '@wordpress/i18n';
 
 function Notice( { className, status, children, onRemove = noop, isDismissible = true } ) {
-	const classNames = classnames( className, 'notice notice-alt notice-' + status, {
+	const classNames = classnames( className, 'components-notice', {
+		[ `is-${ status }` ]: ! ! status,
+	}, {
 		'is-dismissible': isDismissible,
 	} );
 	return (
 		<div className={ classNames }>
-			{ isString( children ) ? <p>{ children }</p> : children }
+			{ isString( children ) ? <div className="components-notice__content">{ children }</div> : children }
 			{ isDismissible && (
-				<button className="notice-dismiss" type="button" onClick={ onRemove }>
+				<button className="components-notice__dismiss" type="button" onClick={ onRemove }>
```

Comment:
Can we replace this by `IconButton` instead of `button`.  I think it already takes care of the scree reader label (label prop) etc...

Reply:
cc @jasmussen for design insights but I'm pretty sure we can override without `!important` (repeating the same specificity, granted that it's not great either)

Reply:
Btw, I don't consider this a blocker for the PR.

Reply:
Yeah, those hover styles are a bit arduous due to having to work with a number of accessibility concerns. However you can override them without !importants, by using the same level of specificity.

For example, if the hover is `.components-icon-button:not(:disabled):_not([aria-disabled="true"]):not(.is-default):hover`, you can override it with `.components-notice__dismiss.components-icon-button:not(:disabled):_not([aria-disabled="true"]):not(.is-default):hover`.

But would also not call that a blocker.

Reply:
Yep. My only concern is that this is less maintainable. If we change the the style rules for `IconButton` component, we might need to also change them in the `Notice` component.

Anyway, I have overridden them with:
```css
&:not(:disabled):not([aria-disabled="true"]):not(.is-default):hover,
&:not(:disabled):not([aria-disabled="true"]):not(.is-default):active,
&:not(:disabled):not([aria-disabled="true"]):focus {
	color: $alert-red;
	background-color: transparent;
}
&:not(:disabled):not([aria-disabled="true"]):not(.is-default):hover {
	box-shadow: none;
}
```

Reply:
Cool — do you really need to override the color though? $alert-red as a hover/active color doesn't seem necessary, and it probably will run into contrast issues. Probably just keep the `color` as is?